integration-testing/src/main/java/io/grpc/testing/integration/Util.java

```diff
@@ -120,4 +130,97 @@ public static void assertEquals(List<? extends MessageLite> expected,
       }
     }
   }
+
+  /**
+   * Builds the SslContext for use by servers
+   *
+   * @return context
+   *
+   * @throws SSLException
+   */
+  public static SslContext buildServerSslContext() throws SSLException {
+    SslProvider provider = null;
+    File trustCertChainFile = null;
+    TrustManagerFactory trustManagerFactory = null;
+
+    File keyCertChainFile = null;
+    File keyFile = null;
+    String keyPassword = null;
+    KeyManagerFactory keyManagerFactory = null;
+
+    Iterable<String> ciphers = null;
+    CipherSuiteFilter cipherFilter = IdentityCipherSuiteFilter.INSTANCE;
+    ApplicationProtocolConfig apn = null;
+    long sessionCacheSize = 0;
+    long sessionTimeout = 0;
+
+    try {
+      keyCertChainFile = Util.loadCert("server1.pem");
+      keyFile = Util.loadCert("server1.key");
+
+      // OpenSSL provider ignores trustCertChainFile?
```

Comment:
It looks like it doesn't... Right now it doesn't support ALPN either, which is a bigger issue.